src/main/java/it/gov/pagopa/pu/migration/model/UploadDetails.java

```diff
@@ -23,6 +23,7 @@ public class UploadDetails extends BaseEntity {
   private Long uploadDetailId;
   @NotNull
   private Long uploadId;
+  private Long organizationId;
```

Comment:
why this field? the organizationId is inside the Upload entity... you cannot have multiple UploadDetail with a different Upload, neither you can have an UploadDetail without Upload